project/Dependencies.scala

```diff
@@ -97,16 +98,13 @@ object Dependencies {
   val thrift            = "org.apache.thrift"          % "libthrift"                          % V.thrift
   val jackson           = "com.fasterxml.jackson.core" % "jackson-databind"                   % V.jackson
 
-  // snowplow: Note jackson-databind 2.14.x is incompatible with Spark
   val badrows           = "com.snowplowanalytics"      %% "snowplow-badrows"                      % V.badrows
   val tracker           = "com.snowplowanalytics"      %% "snowplow-scala-tracker-core"           % V.tracker
   val trackerEmit       = "com.snowplowanalytics"      %% "snowplow-scala-tracker-emitter-http4s" % V.tracker
-  val schemaDdl         = ("com.snowplowanalytics"      %% "schema-ddl"               % V.schemaDdl)
-    .exclude("com.fasterxml.jackson.core", "jackson-databind")
```

Comment:
Don't we need these exclusions anymore ?

Reply:
No, we don't need them.  In a recent previous commit I bumped spark to a newer version, and the newer version is compatible with jackson 2.14.